### DISCUSSION TITLE
Nearby doctor

### DISCUSSION THREAD

**Discussion by Mann54** [scaleracademy/open-source-projects#116] [Ideas]
By making this project patient can take advise from any doctor with connecting online and patient with emergency can also take advantage by best-aid aidvise !

**Comment by whimsyy** [comment]
Best aid advices is not logical as some every health related condition are different and some of them can be critical which can be harm full for the their life.

Can you please elbert in more details like proper list of features properly documented and some workflow of the project to have bright idea of it
